<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToggleTracepointsDialog</class>
 <widget class="QDialog" name="ToggleTracepointsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set debugging output</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="qcb_enableTracing">
     <property name="text">
      <string>Enable tracing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ql_mainCaption">
     <property name="text">
      <string>Available debugging tracepoints</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="ql_filter">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qle_filter"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="qscrArea_tracepoints">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>98</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="qpb_enableAll">
       <property name="text">
        <string>Enable all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qpb_disableAll">
       <property name="text">
        <string>Disable all</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="ql_outputFile">
       <property name="text">
        <string>Output file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qle_outputFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="qpb_chooseOutputFile">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
